packages/create-astro/src/index.ts

```diff
@@ -272,21 +275,66 @@ export async function main() {
 	spinner.succeed();
 	console.log(bold(green('✔') + ' Done!'));
 
+	const installResponse = await prompts({
+		type: 'confirm',
+		name: 'install',
+		message: `Would you like us to run "${pkgManager} install?"`,
+		initial: true,
+	});
+
+	if (!installResponse) {
+		process.exit(0);
+	}
+
+	if (installResponse.install) {
+		const installExec = exec(`cd ${cwd} && ${pkgManager} install`);
+		const installingPackagesMsg = `Installing packages${emojiWithFallback(' 📦', '...')}`;
+		spinner = ora({ color: 'green', text: installingPackagesMsg }).start();
+		await new Promise<void>((resolve, reject) => {
+			installExec.stdout?.on('data', function (data) {
+				// remove newlines from data stream for nicer formatting
+				const oneLiner = data.replace('\n', '');
+				spinner.text = `${installingPackagesMsg}\n${bold(`[${pkgManager}]`)} ${oneLiner}`;
+			});
+			installExec.on('error', (error) => reject(error));
+			installExec.on('close', () => resolve());
+		});
+		spinner.succeed();
+	}
+
 	console.log('\nNext steps:');
 	let i = 1;
 	const relative = path.relative(process.cwd(), cwd);
 	if (relative !== '') {
 		console.log(`  ${i++}: ${bold(cyan(`cd ${relative}`))}`);
 	}
 
-	console.log(`  ${i++}: ${bold(cyan('npm install'))} (or pnpm install, yarn, etc)`);
+	if (!installResponse.install) {
+		console.log(`  ${i++}: ${bold(cyan(`${pkgManager} install`))}`);
+	}
 	console.log(
 		`  ${i++}: ${bold(
 			cyan('git init && git add -A && git commit -m "Initial commit"')
 		)} (optional step)`
 	);
-	console.log(`  ${i++}: ${bold(cyan('npm run dev'))} (or pnpm, yarn, etc)`);
+	const runCommand = pkgManager === 'npm' ? 'npm run dev' : `${pkgManager} dev`;
+	console.log(`  ${i++}: ${bold(cyan(runCommand))}`);
 
 	console.log(`\nTo close the dev server, hit ${bold(cyan('Ctrl-C'))}`);
 	console.log(`\nStuck? Visit us at ${cyan('https://astro.build/chat')}\n`);
 }
+
+function emojiWithFallback(char: string, fallback: string) {
+	return process.platform !== 'win32' ? char : fallback;
+}
+
+/**
+ * Politely stolen from ViteJS
```

Comment:
lol no need for this comment. Vite should already be in our LICENSE! 

I'm always sensitive to this word in open source.

Reply:
update: looks like vite's license was only included inline in a single file. Moved it to our LICENSE file: https://github.com/withastro/astro/blob/main/LICENSE